site/en/_partials/privacy-sandbox/glossary-entries/_README.md

```diff
@@ -0,0 +1,9 @@
+# Using glossary partials
+
+These partials each contain one glossary term. If you use any and need a subhead for them, you'll need to include the subhead inline in your content before the include, like so
+
+    ## Ad exchange
+
+    {% Partial 'privacy-sandbox/glossary-entries/ad-exchange.njk' %}
```

Comment:
This is failing the linter, and should be made a fenced codeblock. https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

````
```txt
## Ad exchange

{% Partial 'privacy-sandbox/glossary-entries/ad-exchange.njk' %}
```
````

Reply:
Ha I originally had the three ticks run inline. My memory failed me! fixed